src/lusty/filesystem-explorer.rb

```diff
@@ -137,6 +137,11 @@ def all_files_at_view
 
       unless @memoized_dir_contents.has_key?(view)
 
+        view_str = view.to_s
+        if view_str.start_with?("scp://")
```

Comment:
Maybe add a method for this in `lusty.rb` since you're doing the same check in a few places?  E.g. `is_scp_path(view_str)`.